### DISCUSSION TITLE
小米软件商店繁体状态下无法跳过开屏广告

### DISCUSSION THREAD

**Discussion by GitHub000666** [AIsouler/GKD_subscription#278] [General]
如题，简体可以，繁体不行。
https://i.gkd.li/i/16323945
![Screenshot_2024-07-23-15-58-16-025_li.songe.gkd.jpg](https://github.com/user-attachments/assets/61f749c3-4a72-468a-b09f-96db47d87d50)

**Comment by AIsouler** [comment]
5bfd23a4444cf5ecf754ff3ff0e05f994d4d3156
